options_linux.cmake

```diff
@@ -85,6 +85,10 @@ if (NOT DESKTOP_APP_USE_PACKAGED OR DESKTOP_APP_SPECIAL_TARGET)
         -fstack-clash-protection
         -fcf-protection
     )
+    if (CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64")
```

Comment:
Please just use target_compile_options_if_exists

Reply:
Sorry, I am new in working with people in github, so I sent you two reviews when I just want to send a single comment. Now I understand the function. What do you think about using -mbranch-protection=pac-ret+bti?

Reply:
> -mbranch-protection=pac-ret+bti

That doesn't seem to be a valid value according to this
![](https://github.com/user-attachments/assets/8a54e393-38e1-42d2-9c3e-affd39005230)

Reply:
So, the default is none. Then what do you think will be the right one, 'standard' or 'pac-ret+leaf+b-key' ?

Reply:
I honestly have no idea

Reply:
I think the right option is -mbranch-protection=pac-ret+leaf as standard doesn't cover leaf functions that can be overwritten to execute code even they don't call other functions, and not using the b-key can be probably safe because using the a-key is enough, while the b-key may be very strong and will have a heavy impact in performance.